src/style.css

```diff
@@ -1 +1,44 @@
 /* add styles here */
+html {
+  margin: 0;
+ }
+
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 16px;
+  width: 16px;
+  margin: 2px;
```

Comment:
Wrong margin, please use the same like in figma 
![image](https://github.com/mate-academy/layout_stars/assets/54936044/61bc4d8f-41f4-4d0e-becc-38182d208278)